src/probnum/quad/__init__.py

```diff
@@ -1,15 +1,28 @@
 """Bayesian Quadrature."""
 
-from ._bayesquad import *
-from .bq_methods import *
+from ._bayesquad import bayesquad
+from ._integration_measures import GaussianMeasure, IntegrationMeasure, LebesgueMeasure
+from .bq_methods import BayesianQuadrature
+from .kernel_embeddings import (
+    KernelEmbedding,
+    _kernel_mean_expquad_gauss,
+    _kernel_mean_expquad_lebesgue,
+    _kernel_variance_expquad_gauss,
+    _kernel_variance_expquad_lebesgue,
+)
+from .policies import sample_from_measure
 
 # Public classes and functions. Order is reflected in documentation.
 __all__ = [
     "bayesquad",
     "BayesianQuadrature",
-    "WarpedBayesianQuadrature",
+    "IntegrationMeasure",
+    "KernelEmbedding",
+    "GaussianMeasure",
+    "LebesgueMeasure",
 ]
 
 # Set correct module paths. Corrects links and module paths in documentation.
 BayesianQuadrature.__module__ = "probnum.quad"
-WarpedBayesianQuadrature.__module__ = "probnum.quad"
+IntegrationMeasure.__module__ = "probnum.quad"
+KernelEmbedding.__module__ = "probnum.quad"
```

Comment:
`GaussianMeasure` and `LebesgueMeasure` should also have their module paths set.

Reply:
Do you mean `probnum.quad._integration_measures` for `IntegrationMeasure`? To my understanding they have been imported to `quad` further up?

Reply:
I meant adding the two lines
```python
GaussianMeasure.__module__ = "probnum.quad"
LebesgueMeasure.__module__ = "probnum.quad"
```
Otherwise I believe the `__module__` might still be set to `probnum.quad._integration_measures`.

Reply:
Oh... haven't had coffee yet :facepalm: